src/components/buttons/buttons.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/no-typos */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue.